README.md

```diff
@@ -58,66 +53,57 @@ ReactDOM.render(
 )
 ```
 
+### Using the client
 
-### Integrating with an existing redux store
-
-`cozy-client` uses redux internally to centralize the statuses of the various fetches and replications triggered by the library, and to store locally the data in a normalized way. If you already have a redux store in your app, you can configure `cozy-client` to use this existing store:
+An instance of `CozyClient` allows you to query and mutate (update) data, here's how it looks:
 
-```js
+```jsx
 import CozyClient from 'cozy-client'
-import { combineReducers, createStore, applyMiddleware } from 'redux'
-import myReducers from './myapp/reducers'
-import myMiddleware from './myapp/middleware'
 
 const client = new CozyClient({
   /*...*/
 })
 
-const store = createStore(
-  combineReducers({ ...myReducers, cozy: client.reducer() }),
-  applyMiddleware(myMiddleware)
+client.query(
+  client.find('io.cozy.todos').where({ checked: false })
+).then(
+  ({ data }) => console.log(data)
 )
 
-ReactDOM.render(
-  <CozyProvider client={client} store={store}>
-    <MyApp />
-  </CozyProvider>,
-  document.getElementById('main')
+client.mutate(
+  client.create('io.cozy.todos', { label: 'Buy bread' })
+).then(
+  ({ data }) => console.log(data.id)
 )
 ```
 
 ## Requesting data
 
-To make it easy to fetch data and make it available to your component, we provide a higher-order component called `connect`. Basic example of usage:
+To make it easy to fetch data and make it available to your component, we provide a Render Props component called `Query`. Basic example of usage:
 
 ```jsx
 import React from 'react'
-import { connect, find } from 'cozy-client'
-
-const query = find('io.cozy.todos').where({ checked: false })
+import { Query } from 'cozy-client'
 
-const TodoList = ({ data, fetchStatus }) =>
-  fetchStatus !== 'loaded'
-    ? <h1>Loading...</h1>
-    : <ul>{data.map(todo => <li>{todo.label}</li>)}</ul>
+const query = client => client.find('io.cozy.todos').where({ checked: false })
 
-export default connect(query)(TodoList)
+const TodoList = () => (
+  <Query query={query}>
+    {({ data, fetchStatus }) =>
+      fetchStatus !== 'loaded'
+        ? <h1>Loading...</h1>
+        : <ul>{data.map(todo => <li>{todo.label}</li>)}</ul>
+    }
+  </Query>
```

Comment:
Wouldn't it be better if we wrote a render `props` instead of a `props.children` callback?

```jsx
const displayData = ({ data, fetchStatus }) =>
  fetchStatus !== "loaded"
    ? <h1>Loading…</h1>
    : <ul>{ data.map(todo => <li>{todo.label}</li>) }</ul>

const TodoList = () => (
  <Query query={query} render={displayData} />
)
```

Reply:
I prefer the latter syntax, but I guess we could handle both.